dtm0/Kbuild.sub

```diff
@@ -3,4 +3,6 @@ m0tr_objects += \
                   dtm0/clk_src.o \
                   dtm0/clk_src_xc.o \
                   dtm0/tx_desc.o \
-                  dtm0/tx_desc_xc.o
+                  dtm0/tx_desc_xc.o \
+		  dtm0/service.o \
+		  dtm0/fop.o
```

Comment:
we need to check the "tabs vs. spaces" issue here. If there are tabs then let's replace them with spaces to be aligned with the rest of the codebase.

Reply:
fixed